data/scripts/bin/update-hosts.sh.template

```diff
@@ -44,20 +44,6 @@ until [[ -n ${cluster_id} ]]; do
 done
 echo "Fetched cluster-id: $cluster_id"
 
-# Register extra manifests, if present. Required only for the interactive workflow.
-if [ "{{.EnableInteractiveFlow}}" = "true" ]; then
```

Comment:
In absence of the interactive flag and with the assumption that appliance will only create an unconfigured ignition, while the saas UI will add the sentinel file /etc/assisted/interactive-ui, does the responsibility of adding the extra manifests goes to UI ( mostly below the sea)? Cc @andfasano